React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,44 @@
+import { isArray } from "./utils";
+
+export function min(...args) {
+  if (args.length == 0) {
+    return undefined;
+  }
+
+  if (isArray(...args)) {
+    var num = Math.min(...args[0]);
+    return num;
+  }
+
+  return Math.min(...args);
+}
+
+export function copy(args) {
+  var example = args;
+
+  if (isArray(example)) {
+    var copieExample = [...example];
+    return [...example], copieExample;
```

Comment:
You don't need to return the value twice. You can just return the copy obtained from the spread operator within the array definition. 🙂